dask/tests/test_threaded.py

```diff
@@ -149,8 +151,8 @@ def test_f():
     assert L == [1] * 20
 
 
-@pytest.mark.slow
-def test_interrupt():
+# This test uses the cleanup fixture to check for leaking threads
```

Comment:
can't you just join on all the threads you explicitly created? I don't think you need to be concerned about leaking from the threadpool.

Reply:
This was an additional safety measure; the pool should be shut down by the context manager. Since this fails mindeps, I'll drop the fixture.